base_layer/key_manager/src/key_manager.rs

```diff
@@ -84,13 +87,17 @@ where
 
     /// Derive a new private key from master key: derived_key=SHA256(master_key||branch_seed||index)
```

Comment:
This comment is no longer accurate, as `SHA-256` is not resistant to length extension attacks, and the input structure implied by the comment is out of date and no longer correct.

Reply:
Absolutely ! Thank you for taking notice of it !